tools/depends/configure.in

```diff
@@ -91,6 +91,39 @@ passed_ldflags="$LDFLAGS"
 passed_cxxflags="$CXXFLAGS"
 platform_cc=gcc
 platform_cxx=g++
+
+case $build in
+  arm*-*linux-android*)
+    build_os="android"
```

Comment:
build_os is... the build os, right?
I very much doubt we'll ever be able to build kodi on android self ;)